Libraries/Components/ScrollResponder.js

```diff
@@ -18,14 +18,16 @@ const ReactNative = require('../Renderer/shims/ReactNative');
 const TextInputState = require('./TextInput/TextInputState');
 const UIManager = require('../ReactNative/UIManager');
 const Platform = require('../Utilities/Platform');
-const ScrollView = require('./ScrollView/ScrollView');
 import Commands from './ScrollView/ScrollViewCommands';
 
 const invariant = require('invariant');
 const performanceNow = require('fbjs/lib/performanceNow');
 
 import type {PressEvent, ScrollEvent} from '../Types/CoreEventTypes';
-import type {Props as ScrollViewProps} from './ScrollView/ScrollView';
+import type {
+  Props as ScrollViewProps,
+  ScrollViewType,
+} from './ScrollView/ScrollView';
```

Comment:
Based on https://flow.org/en/docs/types/modules/#toc-importing-and-exporting-types

I think you can change this to something like:

```
import type ScrollView, {Props as ScrollViewProps} from './ScrollView/ScrollView';
```

And then you don't need to export the type

Reply:
@TheSavior Thank you!
The syntax was a little different, but I was able to use
```jsx
import typeof ScrollView from './ScrollView/ScrollView
import type {Props as ScrollViewProps } from './ScrollView/ScrollView'
```